setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup
```

Comment:
What does this file do?

Reply:
Identifies the script: https://click.palletsprojects.com/en/7.x/setuptools/